### PR TITLE
appease our rubocop overlords

### PR DESCRIPTION
**Pull request by ndushay**
## Why was this change made? 🤔

to make this repo more consistent with our other repos, and easier to understand the specs.  Part of #194

Most of the changes are autocorrect.  I've tried to indicate where that is NOT true.

## How was this change tested? 🤨

⚡ ⚠ If this change has cross service impact (e.g. changes what is written to filesystem, or what is expected to be read from filesystem), run ***[integration test create_preassembly_image_spec.rb](https://github.com/sul-dlss/infrastructure-integration-test)*** on stage as it tests preservation, and/or test in stage environment, in addition to specs.⚡

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -25,7 +25,7 @@ RuboCop::RakeTask.new
 
 task :clean do
   puts 'Cleaning old coverage'
-  FileUtils.rm('coverage.data') if File.exist?('coverage.data')
```
Comment (ndushay) [inline Rakefile:28]: this was autocorrect

File: lib/stanford/content_inventory.rb [hunk 2]
```diff
@@ -86,7 +86,7 @@ def generate_instance(node)
       instance.path = node.attributes['id'].content
       instance.datetime = begin
         node.attributes['datetime'].content
-      rescue
+      rescue StandardError
```
Comment (ndushay) [inline lib/stanford/content_inventory.rb:89]: adding StandardError to this seems fine.

File: spec/features/inventory/write_inventory_xml_spec.rb [hunk 3]
```diff
@@ -1,25 +1,25 @@
...
-    xmlObj1 = Nokogiri::XML(inventory_pathname.read)
-    xmlObj1.xpath('//@datetime').each { |d| d.value = '' }
-    xmlObj1.xpath('//@inventoryDatetime').remove
+    xml_object1 = Nokogiri::XML(inventory_pathname.read)
+    xml_object1.xpath('//@datetime').each { |d| d.value = '' }
+    xml_object1.xpath('//@inventoryDatetime').remove
     output_dir.rmtree
 
-    xmlTest = <<-XML
+    expected_xml = <<-XML
```
Comment (ndushay) [inline spec/features/inventory/write_inventory_xml_spec.rb:22]: this is clearer this way, IMO.

File: spec/features/serializer/write_xml_spec.rb [hunk 4]
```diff
@@ -1,23 +1,23 @@
...
     catalog_object.write_xml_file(output_dir)
     catalog_pathname = output_dir.join('signatureCatalog.xml')
-    xmlObj1 = Nokogiri::XML(catalog_pathname.read)
-    xmlObj1.xpath('//@catalogDatetime').remove
+    xml_object1 = Nokogiri::XML(catalog_pathname.read)
+    xml_object1.xpath('//@catalogDatetime').remove
     output_dir.rmtree
 
-    catalog_test = <<-XML
+    expected_xml = <<-XML
```
Comment (ndushay) [inline spec/features/serializer/write_xml_spec.rb:20]: similarly, I think this is clearer too

File: spec/unit_tests/moab/file_inventory_difference_spec.rb [hunk 5]
```diff
@@ -40,7 +40,7 @@
       expect(diff_v1_v2).to be_instance_of(described_class)
     end
 
-    context 'sets attributes' do
+    describe 'attributes set' do
```
Comment (ndushay) [inline spec/unit_tests/moab/file_inventory_difference_spec.rb:43]: still not in love with this, but it was a decent attempt to be clearer without a lot of effort on the individual tests ...

**Review by ndushay — COMMENTED** [review]
I've tried to call out where any significant judgement was exercised...

a great deal of the changes are StringLiteral (quote style);  I can split that into a separate commit if that's helpful.

File: .rubocop.yml [hunk 6]
```diff
@@ -44,25 +44,17 @@ RSpec/NestedGroups:
...
-# because ' vs " isn't a big deal for readability or maintainability or execution time
-Style/StringLiterals:
-  Enabled: false
-
 Style/ZeroLengthPredicate:
   Exclude:
     - 'lib/moab/bagger.rb' # is calling `x.size == 0` on Pathname objects, which don't have `.empty?` on old Rubies
 
 RSpec/ExampleLength:
   Exclude:
```
Comment (justinlittman) [inline .rubocop.yml:56]: Why not just `Enabled: false`?

**Review by justinlittman — APPROVED** [review]
Approve pending consideration of comment.

File: .rubocop.yml (hunk 6 above)
Comment (ndushay) [reply .rubocop.yml]: thank you - that's a better way.

**Pull request merged**